getting-started/deployment_guide.md

```diff
@@ -5,6 +5,40 @@ This guide assumes you have access to a [k8s cluster](support_matrix.md). If you
 Check the [KubeArmor support matrix](support_matrix.md) to verify if your platform is supported.
 
 ## Install KubeArmor
+
+<details>
+<summary><strong>On MicroShift</strong></summary>
```

Comment:
@anurag-rajawat Is there any way these steps can be done automatically through snitch/operator?
Also, if we can't even install snitch without "kubearmor" namespace having these labels, should we give this configuration through Helm charts? cc @daemon1024

Reply:
There are two things we need to take care of:
1. `kubearmor` **namespace:** I can think of two possible ways to deal with this:

     > Also, if we can't even install snitch without "kubearmor" namespace having these labels, should we give users the ability to configure these with a value in Helm charts?


    - We can use the `--create-namespace --set labels.<labels>` flags while installing like

       ```shell
       helm upgrade --install kubearmor-operator kubearmor/kubearmor-operator -n kubearmor --create-namespace --set labels.security.openshift.io/scc.podSecurityLabelSync="false",labels.pod-security.kubernetes.io/enforce=privileged,labels.pod-security.kubernetes.io/audit=privileged,labels.pod-security.kubernetes.io/warn=privileged
       ```
    - We can create a template for the namespace like

      ```yaml
      apiVersion: v1
      kind: Namespace
      metadata:
        name: kubearmor
        {{- if .Values.environment.isMicroshift }}
        labels:
          security.openshift.io/scc.podSecurityLabelSync: "false"
          pod-security.kubernetes.io/enforce: privileged
          pod-security.kubernetes.io/audit: privileged
          pod-security.kubernetes.io/warn: privileged
        {{- end }}
      ```
      then we need to generate the template and then apply it like
      ```shell
      helm template kubearmor-operator kubearmor/kubearmor-operator -n kubearmor | kubectl apply -f -
      ```
      This will make uninstallation a bit hard. i.e., to uninstall it we need to use
      ```shell
      helm template kubearmor-operator kubearmor/kubearmor-operator -n kubearmor | kubectl delete -f -
      ```
3. `kubearmor-scc` SCC
> Is there any way these steps can be done automatically through snitch/operator?

Without SCC snitch can not run and [SCC](https://docs.openshift.com/container-platform/4.13/authentication/managing-security-context-constraints.html#security-context-constraints-creating_configuring-internal-oauth) is a global resource and it should be created only by cluster-admin that's why we are not creating it as part of the installation.
https://kubearmor.slack.com/archives/C01F9V3SEHY/p1697559969927649


Please let me know if you have any other suggestions on how to handle these issues.